server/core/src/config.rs

```diff
@@ -1112,8 +1112,13 @@ impl ConfigurationBuilder {
                 client_ca,
             }),
             _ => {
-                eprintln!("ERROR: Tls Private Key and Certificate Chain are required.");
-                return None;
+                match bindaddress {
+                    Some(ref bindaddress) if bindaddress.starts_with("/") => None,
+                    _ => {
+                        eprintln!("ERROR: Tls Private Key and Certificate Chain are required.");
```

Comment:
Minor nit while we're here. :smile:

```suggestion
                        eprintln!("ERROR: TLS Private Key and Certificate Chain are required.");
```